src/keri/core/eventing.py

```diff
@@ -1695,14 +1700,16 @@ def locallyDelegated(self, pre: str):
         of self. Unaccepted dip events do not have self.delpre set yet.
 
         Returns:
-            (bool): True if pre is local hab but not group hab
+            (bool): True if pre is local hab or group hab that has a local member
                         When pre="" empty or None then returns False
 
         Parameters:
             pre (str): qb64 identifier prefix if any.
         """
+
         pre = pre if pre is not None else ""
-        return self.locallyOwned(pre=pre)
+        return pre in self.prefixes or pre in self.groups
```

Comment:
I thought `self.prefixes` also includes `self.groups`, so the `or` clause is redundant?

Reply:
Yes your correct, my bad.